packages/config/config/devices/0x0039/39339_zw3107.json

```diff
@@ -1,35 +1,52 @@
-// Honeywell 39339 (ZW3107)
+// Honeywell 39339 / 39446 / ZW3107
 // Z-Wave Plug-in Smart Dimmer
 {
 	"manufacturer": "Honeywell",
 	"manufacturerId": "0x0039",
-	"label": "39339 (ZW3107)",
+	"label": "39339 / 39446 / ZW3107",
```

Comment:
I'm not sure if we should group devices with a different label just becausue they have the same parameters.
That could lead to confusion.

Reply:
Seems to be the only occurence of this though.

Reply:
These share a product type/id so no option but to group them.